pom.xml

```diff
@@ -435,7 +428,7 @@
     <version.org.kie.workbench.app>${version.org.kie}</version.org.kie.workbench.app>
 
     <version.thorntail>2.7.0.Final</version.thorntail>
-    <version.net.byte-buddy>1.10.3</version.net.byte-buddy>
+    <version.net.byte-buddy>1.9.11</version.net.byte-buddy>
```

Comment:
not sure this downgrade is ok, see why it was upgraded before as Mockito tests were failing with this older version https://github.com/kiegroup/droolsjbpm-build-bootstrap/commit/0ebdb10c3ddb06f371e91b6edc04d8aae101a900

Reply:
Hello @mareknovotny  I reverted the downgrade in 1dd4fcf7849e8eeb3b927b89384286e546c77c74 - Thanks!